rfcs/accordians-component.md

```diff
@@ -0,0 +1,141 @@
+        ]}
+        expanded={[false, true]}
+        autoExpand={true}
+      />
+    </div>
+  );
+};
+```
+
+## Exports
+
+* `Accordians`
+* `StyledRoot`
+* `StyledToggle`
+* `StyledTitle`
+* `StyledContent`
+
+## `Accordians` API
+
+* `accordians: Array<Item>` - Required
+  All accordians in component. Each `Item` has:
+  * `title: React$Node` - title of accordian
+  * `content: React$Node` - content of accordian
+* `autoExpand: boolean` - Optional
+  If set to true, it will auto expand next accordian right after current is collapsed. Default is `'false`
+* `expanded: Array<boolean>` - Optional
+  Array of the same size as `accordians`. If element is `true` corresponding accordian is expanded. Default is `Array<false>`
+* `disabled: boolean`:
+  Disable control from being changed  
+* `overrides: {Root: (props: {[string]: any}) => React$Node, Toggle: (props: {[string]: any}) => React$Node, Title: (props: {[string]: any}) => React$Node, Content: (props: {[string]: any}) => React$Node}`
+  * `Root` container element to render.
+  * `Toggle` is the expand\collapse button for each accordian.
+  * `Title` to render for each accordian.
+  * `Content` to render for each accordian.
+* `onToggle: func(event, index, isExpand)` - Optional
+  handler for events on trigger element. `index` indicates number of accordian toggled. `isExpand` boolean to show if accordian expands or collapses.
+
+## `StatefulAccordians` API
+
+* `initialState: {}`
+  Initial state of an uncontrolled accordians component.
+  * `expanded: Array<boolean>` initial state of array of the same size as `accordians`. If element is `true` corresponding accordian is expanded
+* `stateReducer: (type: text, nextState: {}, currentState: {}, e: any) => nextState`
+  A state change handler.
+  * `type` - state change type
+  * `nextState` - a new state changes that will be set
+  * `currentState` - current full state of the component
+* `accordians: Array<Item>` - Required
+  All accordians in component. Each `Item` has:
+  * `title: React$Node` - title of accordian
+  * `content: React$Node` - content of accordian
+* `autoExpand: boolean` - Optional
+  If set to true, it will auto expand next accordian right after current is collapsed. Default is `'false`  
+* `disabled: boolean` - Optional
+  Disable control from being changed
+* `expanded: Array<boolean>` - Optional
+  Array of the same size as `accordians`. If element is `true` corresponding accordian is expanded. Default is `Array<false>`
+* `onToggle: func(event, index, isExpand)` - Optional
+  handler for events on trigger element. `index` indicates number of accordian toggled. `isExpand` boolean to show if accordian expands or collapses.
+
+## Presentational components props API
+
+These properties are passed to every presentational (styled) component that is exported:
+
+* `$disabled: boolean`
+  Disable control from being changed
+* `$isHovered: boolean`
+  If component is hovered
+* `$isActive: boolean`
+  If component is active
```

Comment:
What does it mean for the Accordion to be "Active"?

Reply:
removed. no meaning